flutter/lib/src/default_integrations.dart

```diff
@@ -221,6 +221,15 @@ class NativeSdkIntegration extends Integration<SentryFlutterOptions> {
       );
     }
   }
+
+  @override
+  FutureOr<void> close() async {
+    try {
+      await _channel.invokeMethod<void>('closeNativeSdk');
+    } catch (error) {
+      // TODO what to do?
```

Comment:
@marandaneto Currently we can't log something in the `Integration.close` emthods because `SentryOptions` are passed to `Integration.call`. They aren't an instance property.

Do we need to log this error? If yes I would add SentryOptions as an instance property.

Reply:
you mean `options.logger` does not work? maybe it's something we should fix across the SDK then, if it's the case, let's raise a new issue for that, we don't need to tackle within this PR.

Reply:
It does work, but I don't have access to it in the close method. So my question is should we add `SentryOptions` as a parameter to the `Integration.close` method. Or just pass `SentryOptions` as a constructor parameter.

Reply:
you get the `SentryFlutterOptions` instance when the method `call` is called, save it to a global field (of the integration), or take it as a ctor param, both would work